@xen-orchestra/backups/RemoteAdapter.js

```diff
@@ -487,7 +486,6 @@ class RemoteAdapter {
         },
       })
     } else {
-      path = `${basePath}/${basename}.vhd`
       this.outputStream(path, input, { checksum, validator })
```

Comment:
Should it be `path` or `dataPath`?

Reply:
I know they are the same in this case, but what would make the more sense? :thinking:

Reply:
it should be datapath, if we use an alias to a vhd file , you're right